src/main/kotlin/gg/essential/essential-loader.gradle.kts

```diff
@@ -0,0 +1,69 @@
+package gg.essential
+
+import gg.essential.gradle.multiversion.Platform
+import gg.essential.gradle.util.RelocationTransform.Companion.registerRelocationAttribute
+
+
+plugins {
+    id("gg.essential.loom")
+}
+
+val platform = Platform.of(project)
+
+val essentialLoader: Configuration by configurations.creating
+
+when {
+    platform.isLegacyForge -> {
+        dependencies {
+            "implementation"(essentialLoader("gg.essential:loader-launchwrapper:1.2.2")!!)
```

Comment:
Maybe however the original intent was for the dev to have to do as little as possible (and I think it's mostly fine that the version is hardcoded since in theory the stage0 version should almost never change)